server/migrations.js

```diff
@@ -80,7 +80,7 @@ Migrations.add('lowercase-board-permission', () => {
 Migrations.add('change-attachments-type-for-non-images', () => {
   const newTypeForNonImage = 'application/octet-stream';
   Attachments.find().forEach(file => {
-    if (!file.isImage()) {
+    if (!file.isImage) {
```

Comment:
Is this change intended?

Reply:
Yes, `isImage` is an attribute in `Meteor-Files` rather than a method in `Collection-FS`.

Reply:
Is this going to work if, for example, someone is trying to update from a really old version of Wekan to a newer one?

Reply:
In that case, the `server/migration.js` should have updated all records when starting up so it should work.

I only tested with a not-old version of Wekan (current running version), though.

Reply:
Oh, no, sorry, I misunderstood this. I think this modification is an error when I fixing around `isImage()` to `isImage`. I'll recover this.